apps/condo/domains/contact/access/Contact.js

```diff
@@ -29,12 +30,11 @@ async function canReadContacts (args) {
         return await canReadObjectsAsB2BAppServiceUser(args)
     }
 
+    const permittedOrganizations = await getEmployedOrRelatedOrganizationsByPermissions(user, 'canReadContacts')
+
     return {
         organization: {
-            OR: [
-                queryOrganizationEmployeeFor(user.id, 'canReadContacts'),
-                queryOrganizationEmployeeFromRelatedOrganizationFor(user.id, 'canReadContacts'),
-            ],
+            id_in: permittedOrganizations,
```

Comment:
Could you please check that if the ‘ permittedOrganizations’ is empty we will not have any access.

Reply:
1. It's checked by many tests where employee with some permission can and without cannot
2. Empty `id_in` works as you expect it to work

<img width="1477" alt="image" src="https://github.com/open-condo-software/condo/assets/50069872/655c1106-a767-40c2-8899-b0d81248c4e5">
<img width="1477" alt="image" src="https://github.com/open-condo-software/condo/assets/50069872/d617ce62-e820-40fd-96af-e9a2445e88cc">